lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -160,16 +138,8 @@ def support_files_temp_dir
           # @return [void]
           #
           def create_info_plist_file(path, native_target, version, platform, bundle_package_type = :fmwk)
-            UI.message "- Generating Info.plist file at #{UI.path(path)}" do
-              generator = Generator::InfoPlistFile.new(version, platform, bundle_package_type)
-              update_changed_file(generator, path)
-              add_file_to_support_group(path)
-
-              relative_path_string = path.relative_path_from(sandbox.root).to_s
-              native_target.build_configurations.each do |c|
-                c.build_settings['INFOPLIST_FILE'] = relative_path_string
-              end
-            end
+            TargetInstallerHelper.create_info_plist_file(@sandbox, path, native_target, version, platform, bundle_package_type)
+            add_file_to_support_group(path)
```

Comment:
note here: `add_file_to_support_group` is only needed when `create_info_plist_file` is called by subclasses of `PodTargetInstaller` and extracting that function out to `TargetInstallerHelper` would require more changes that feel excessive, so this is my workaround there. is this an acceptable compromise